ocaml-stdlib/ocaml-stdlib.tex

```diff
@@ -187,7 +187,7 @@ \subsection{module Map}
 let dict = {@vb{}Dict.add} "x" value_x empty
 if {@vb{}Dict.mem} "x" dict then ...
 let value_x = {@vb{}Dict.find} "x" dict (* Not_found *)
-let value_x_o = {@vb{}Dict.find@_opt} "x" dict
+let value_x_o = {@vb{}Dict.find@_opt} "x" dict (* None *)
```

Comment:
I disagree with that one, the convention for this sheet was to only put exception in comments. the `_o` and the `_opt` make it clear enough imo.

Reply:
Well, immediatly seeing the 2 differences (instead of "thinking" _opt -> option -> None)
seems easier, but... why not!

Comment deleted.